events/index.js

```diff
@@ -1,9 +1,11 @@
 'use strict'
 
+// Load events handlers here
 const welcomeTask = require('./welcome');
```

Comment:
Should we rename `event` to `eventHandler`? Also `welcomeTask` should be renamed to `welcomeEvent(Handler)`